app/jobs/patient_list_download_job.rb

```diff
@@ -1,14 +1,21 @@
-class PatientListDownloadJob < ApplicationJob
+class PatientListDownloadJob
+  include Sidekiq::Worker
+
+  sidekiq_options lock: :until_executed,
```

Comment:
So this lock is for subsequent jobs that are queued with the exact same parameters? If two different people request a facility's linelist, they'd both get it right?

Reply:
Yes. Tested this.